images/kubekins-e2e/Dockerfile

```diff
@@ -93,9 +93,11 @@ RUN if [ -n "${KIND_VERSION}" ]; then \
     fi
 
 # install kubetest2 binaries if a version is provided
+# kubetest2 must be installed from git as some important build flags are set by make
 ARG KUBETEST2_VERSION
 RUN if [ -n "${KUBETEST2_VERSION}" ]; then \
-    go install sigs.k8s.io/kubetest2/...@"${KUBETEST2_VERSION}"; \
```

Comment:
this installed an specific version but now we install only master

Reply:
In the cloud-build file, it was pinned to latest. So this change affect us. Also, kubetest2 isn't versioned yet. Something to fix later this year.

kubetest2 has robust e2e testing now that has to pass

Reply:
you can do

```
go install sigs.k8s.io/kubetest2/...@84d26e273777e0ce54175d71211c783dc9678150
go: downloading sigs.k8s.io/kubetest2 v0.0.0-20230731212155-84d26e273777
go: downloading google.golang.org/api v0.115.0
go: downloading k8s.io/release v0.15.1
go: downloading sigs.k8s.io/boskos v0.0.0-20230524062849-a7ef97ee445d
go: downloading k8s.io/apimachinery v0.27.4
```

Reply:
this should be something like this then
```
git clone --depth=1 --branch ${KUBETEST2_VERSION}  https://github.com/kubernetes-sigs/kubetest2.git 
```